docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -0,0 +1,416 @@
+
+## Converter registration precedence
+
+The `Newtonsoft.Json` registration precedence for custom converters is as follows:
+
+* Attribute on property
+* Attribute on type
+* `Converters` collection
+
+This order means that a custom converter in the `Converters` collection is overridden by a converter that is registered by applying an attribute at the type level. Both of those registrations are overridden by an attribute at the property level.
+
+The `System.Text.Json` registration precedence for custom converters is different:
+
+* Attribute on property
+* `Converters` collection
+* Attribute on type
+
+The difference here is that a custom converter in the `Converters` collection overrides an attribute at the type level. The intention behind this order of precedence is to make run-time changes override design-time choices. There's no way to change the precedence.
+
+## `Newtonsoft.Json` [JsonProperty] attribute
+
+`Newtonsoft.Json` has a `[JsonProperty]` attribute that is used for a variety of functions. `System.Text.Json` has single-purpose attributes instead of a multi-purpose attribute. The following attributes serve purposes similar to some of the `Newtonsoft.Json` attribute functions:
+
+* [[JsonPropertyName]](system-text-json-how-to.md#customize-individual-property-names)
+* [[JsonConverter]](system-text-json-converters-how-to.md#register-a-custom-converter)
+
+Other functions of the `Newtonsoft.Json` `[JsonProperty]` attribute have no equivalent attribute in `System.Text.Json`. For some of those functions, there are other ways to accomplish the same purpose, as detailed in the following sections of this article.
+
+## Character escaping
+
+During serialization, `Newtonsoft.Json` is relatively permissive about letting characters through without escaping them. (That is, it doesn't replace them with `\uxxxx` where `xxxx` is the Unicode code of the character.) `System.Text.Json` escapes more characters by default to provide defense-in-depth protections against cross-site scripting (XSS) or information disclosure attacks. `System.Text.Json` escapes all non-ASCII characters by default, so you don't need to do anything if you're using `StringEscapeHandling.EscapeNonAscii`. For information about how to override the default `System.Text.Json` behavior, see [Customize character encoding](system-text-json-how-to.md#customize-character-encoding).
+
+## Specify date format
+
+`Newtonsoft.Json` provides a `DateTimeZoneHandling` option that lets you specify the serialization format used for `DateTime` and `DateTimeOffset` properties. In `System.Text.Json`, the only format that has built-in support is ISO 8601-1:2019. To use any other format, create a custom converter. For more information, see [DateTime and DateTimeOffset support in System.Text.Json](../datetime/system-text-json-support.md).
```

Comment:
We should also mention `DateFormatString` here since that is folks would use when parsing non-ISO dates with `Newtonsoft.Json:
https://www.newtonsoft.com/json/help/html/P_Newtonsoft_Json_JsonSerializerSettings_DateFormatString.htm

`DateTimeZoneHandling` is more about how the datetime is serialized/deserialized when the JSON strings are according to ISO. 

We should take some inspiration from https://www.newtonsoft.com/json/help/html/DatesInJSON.htm